src/access/acp.test.ts

```diff
@@ -132,7 +140,59 @@ describe("hasInaccessiblePolicies", () => {
     expect(internal_hasInaccessiblePolicies(resourceWithAcr)).toEqual(false);
   });
 
-  it("returns false if the ACR only contains reference to Rules in the same Resource", () => {
+  it("returns true if the ACR references an active Policy in a different Resource", () => {
+    const plainResource = mockSolidDatasetFrom("https://some.pod/resource");
+    const resourceWithAcr = addMockAcrTo(
+      plainResource,
+      mockAcr("https://some.pod/resource", {
+        policies: ["https://some.pod/anoter-resource?ext=acr#policy"],
```

Comment:
Small typo, also present in the other test:

```suggestion
        policies: ["https://some.pod/another-resource?ext=acr#policy"],
```

Reply:
Resolved in 611a7c6a4a79997029573d28feaef0c8ef75523d